script/generate-e2e-tests.js

```diff
@@ -1556,20 +1558,28 @@ for (const [component, info] of components) {
     });
   }
 })`
-  })
+      })
 
-  const source = recast.parse(`import {test, expect} from '@playwright/test'
+      const source = recast.parse(`import {test, expect} from '@playwright/test'
 import {visit} from '../test-helpers/storybook'
 import {themes} from '../test-helpers/themes'
 
 test.describe('${component}', () => {
   ${stories.join('\n\n')}
 })`)
 
-  const {code} = recast.print(source)
-  const formatted = prettier.format(code, {
-    parser: 'typescript',
-    ...prettierConfig,
-  })
-  fs.writeFileSync(filepath, formatted, 'utf8')
+      const {code} = recast.print(source)
+
+      const formatted = await prettier.format(code, {
+        parser: 'typescript',
+        ...prettierConfig,
+      })
+      fs.writeFileSync(filepath, formatted, 'utf8')
+    }
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error)
+  }
 }
+
+generateE2ETests()
```

Comment:
One suggestion, instead of the try-catch block we could instead add `.catch()` to the promise call and log/exit from there.

```suggestion
generateE2ETests()
  .catch((error) => {
    // eslint-disable-next-line no-console
    console.log(error)
    process.exit(1)
  });
```